src/components/Comments/Comments.jsx

```diff
@@ -21,11 +24,40 @@ const Comments = ({article_id}) => {
         })
     },[])
 
+    const handleChange = (event) => {
+        setNewComment(event.target.value);
+    }
+
+    const postComment = () => {
+        setIsPostingComment(true);
+        setPosted(false);
+        if (!newComment){
+            console.log('no')
+            return;
+        }
+        requests.addComment(article_id, {"username":user, "body":newComment})
+        .then((data) => {
+            setComments([data,...comments]);
+            setIsPostingComment(false);
+            setPosted(true);
+        })
+        .catch((err) => {
+            if(err.code === "ERR_NETWORK"){
+                alert('Internet connection is offline..')
```

Comment:
I'm glad you're looking at error messages here, but an alert is not the most userfriendly way of showing there is an issue. On mobile these do not display the same way as they do on desktop. Consider another way of getting this message across.